controllers/admin/AdminPreferencesController.php

```diff
@@ -194,6 +194,13 @@ public function __construct()
                     'cast' => 'intval',
                     'type' => 'bool',
                 ),
+                'PS_DISPLAY_MANUFACTURERS' => array(
+                    'title' => $this->trans('Display brands', array(), 'Admin.Shopparameters.Feature'),
+                    'desc' => $this->trans('Enable brands page on your front office even when its module are disabled.', array(), 'Admin.Shopparameters.Help'),
```

Comment:
```suggestion
                    'desc' => $this->trans('Enable brands page on your front office even when its module is disabled.', array(), 'Admin.Shopparameters.Help'),
```

Reply:
We write this element twice? cf. https://github.com/PrestaShop/PrestaShop/pull/14665/files#diff-bc4c4faa966c237b7346ad189fdda6de

Reply:
yes, but it will not be displayed twice on the page, I just followed the existing logic for other fields in the same page and gave the translation key in this place as well.